android-components/components/support/ktx/src/main/java/mozilla/components/support/ktx/kotlin/String.kt

```diff
@@ -196,7 +195,12 @@ fun String.sha1(): String {
     return digest.joinToString(
         separator = "",
         transform = { byte ->
-            String(charArrayOf(characters[byte.toInt() shr 4 and 0x0f], characters[byte.toInt() and 0x0f]))
+            String(
+                charArrayOf(
+                    characters[byte.toInt() shr 4 and 0x0f],
+                    characters[byte.toInt() and 0x0f],
+                ),
+            )
```

Comment:
nit: Lets try to avoid adding unrelated changes :)

Reply:
I missed this autoformat from AndroidStudio, thanks for spotting it!